CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -1040,8 +1040,29 @@ local function CreateSelector(selectionStringTable, startPosition)
 			local isSelected = (i == index)
 
 			if not selectionLabel:IsDescendantOf(game) then
-				this.CurrentIndex = i
-				indexChangedEvent:Fire(index)
+				for i,selectionLabel in pairs(this.Selections) do	
```

Comment:
I'd prefer to keep a tweening update outside the scope of a bugfix, unless this fixes something I'm not aware of.